corehq/apps/couch_sql_migration/casediff.py

```diff
@@ -336,6 +336,7 @@ def diff_form_state(form_id, *, in_couch=False):
     couch_miss = "missing"
     if not in_couch and get_blob_db().metadb.get_for_parent(form_id):
         couch_miss = MISSING_BLOB_PRESENT
+        log.warning("couch form missing, blob present: %s", form_id)
```

Comment:
This is not a great solution for the problem that https://github.com/dimagi/commcare-hq/commit/309fecbf7b084a27cb70e85e845aae9ad86cef3f is trying to solve because it will be done in subprocesses where it is unclear if logging output will be reliably routed to a file. I'm still thinking about how to better solve this issue.